pkg/providers/cloudstack/template.go

```diff
@@ -198,11 +198,15 @@ func buildTemplateMapCP(clusterSpec *cluster.Spec) (map[string]interface{}, erro
 		"eksaSystemNamespace":                        constants.EksaSystemNamespace,
 	}
 
-	auditPolicy, err := common.GetAuditPolicy(clusterSpec.Cluster.Spec.KubernetesVersion)
-	if err != nil {
-		return nil, err
+	if clusterSpec.Cluster.Spec.ControlPlaneConfiguration.AuditPolicyContent != "" {
+		values["auditPolicy"] = clusterSpec.Cluster.Spec.ControlPlaneConfiguration.AuditPolicyContent
```

Comment:
do we want to do a `strings.TrimSpace` whenever we retrieve audit policy from cluster config? I think yaml.Unmarshal will consider string with leading/trailing as valid yaml, but it might be better to remove it